src/compas/geometry/brep/brep.py

```diff
@@ -0,0 +1,1011 @@
+from compas.geometry import Geometry
+from compas.plugins import pluggable
+from compas.plugins import PluginNotInstalledError
+
+
+LINEAR_DEFLECTION = 1e-3
+
+
+@pluggable(category="factories")
+def new_brep(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_brep(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_mesh(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_box(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_cylinder(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_sphere(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_cone(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_surface(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_torus(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_sweep(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_step_file(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_polygons(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+@pluggable(category="factories")
+def from_curves(*args, **kwargs):
+    raise PluginNotInstalledError()
+
+
+class BrepType:
```

Comment:
i think, in Rhino (py 2.7) this will result in an "old style" class.
should be `class BrepType(object)`.
however, not sure this matters here...